ui/shared/DropdownMenuItems.jsx

```diff
@@ -4,12 +4,19 @@ import { DropdownMenu, DropdownItem } from 'reactstrap';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faCheck } from '@fortawesome/free-solid-svg-icons';
 
-const createDropdownItem = (item, namespace, updateData, selectedItem) => {
+const createDropdownItem = (
+  item,
+  namespace,
+  updateData,
+  selectedItem,
+  className,
+) => {
   return (
     <DropdownItem
       tag="a"
       key={`${namespace}${item}`}
       onClick={() => updateData(item)}
+      className={`${!className || ''}`}
```

Comment:
I think the tests will pass if we change this line to `className={className || ''}`